package.json

```diff
@@ -30,7 +30,7 @@
     "LICENSE",
     "PATENTS",
     "README.md",
-    "flow/",
```

Comment:
This was right - `flow/include` is supposed to get shipped to npm so it can get consumed by packages depending on fbjs and using flow (eg, Relay). https://github.com/facebook/relay/blob/master/src/.flowconfig#L8.

We're waiting on the flow team for a better solution but need this in the meantime.